publications/index.md

```diff
@@ -4,4 +4,14 @@ classes: wide
 author_profile: true
 ---
 
-## Publications
\ No newline at end of file
+## Publications
+
+Chen, Yinlin, and James Tuttle. *Virginia Tech Libraries’ Next Gen Digital Libraries Platform*. Jan. 2020, [hdl.handle.net/10919/96332](http://hdl.handle.net/10919/96332). PowerPoint Presentation.
```

Comment:
Chen, Yinlin, and James Tuttle. "Virginia Tech Libraries’ Next Gen Digital Libraries Platform."  IT Council Symposium 2020, 8 January, 2020, Blacksburg, VA, [hdl.handle.net/10919/96332](http://hdl.handle.net/10919/96332). Conference Presentation.